src/status_im/ui/screens/communities/profile.cljs

```diff
@@ -51,7 +51,8 @@
                description]
               [quo/separator {:style {:margin-vertical 8}}]])
            [:<>
-            (let [link (communities/universal-link community-id)]
+            (let [link #_{:clj-kondo/ignore [:unresolved-var]}
```

Comment:
Just out of curiosity, did you have to disable the linter on this? it looks like it's being used, so the linter should not complain

Reply:
Thanks. This was used before by me and was left by mistake.